src/main/webapp/app/shared/profile-picture/profile-picture.component.html

```diff
@@ -3,7 +3,7 @@
         <strong
             [id]="defaultPictureId()"
             class="profile-picture rounded-3"
-            [ngStyle]="{ 'background-color': profilePictureBackgroundColor, 'font-size': fontSize }"
+            [ngStyle]="{ 'background-color': isGray() ? '#888' : profilePictureBackgroundColor, 'font-size': fontSize }"
```

Comment:
Our guidelines:
> Do not hard code any color values in your component stylesheets, templates or TypeScript files!

https://docs.artemis.cit.tum.de/dev/guidelines/client-design#color-usage

Reply:
Changed the gray to use grayscale instead of hard coded inline color in this [commit](https://github.com/ls1intum/Artemis/pull/9624/commits/82a35634c5635a96949b32342c0bee087f8da25a).